libs/core/src/lib/multi-input/multi-input.component.ts

```diff
@@ -309,6 +310,9 @@ export class MultiInputComponent implements
                 this.buildComponentCssClass();
             })
         );
+        if (!this.inputId) {
+            this.inputId = uuidv4();
```

Comment:
Why not use it as a default value on line 115?

Reply:
let's move it on the line 115 as initial value

Reply:
@N1XUS @dimamarksman For performance. When we move value to 115 line, uuidv4 will invoke for every case.
On this line, uuidv4 invoke only when user doesn't pass inputId.